server/controllers/pages/cart.js

```diff
@@ -0,0 +1,13 @@
+const { getItems } = require('../../database/queries/cart');
+
+const getCart = (req, res) => {
+  const { userId } = req.cookies;
+  getItems(userId)
+    .then((data) => {
+      res.json(data);
+    })
+    .catch((err) => {
+        console.log(err.message);
+        res.status(500).json({ message: 'Internal server error' });
```

Comment:
remove console, and you can just `next(err)` which will direct you to the **500** error controller